frontend/resourceadm/types/global.ts

```diff
@@ -1,10 +1,8 @@
 export type NavigationBarPage = 'about' | 'policy' | 'deploy' | 'migration';
 
```

Comment:
I know it is not related to these changes, but we are now trying to create separate files for the different related types using more meaningful names than ```global.ts```, might be worth changing it here in the future too 😄